### PR TITLE
[MeshMap] Added BETA superscript to page title 

### LINKED ISSUE #2515: [MeshMap] Add "BETA" superscript to page title
**Current Behavior**
The [MeshMap beta sign up page](https://layer5.io/service-mesh-management/meshmap) has "MeshMap" as its title.

**Desired Behavior**
Add the word "BETA" as superscript to "MeshMap", like so (except using black font color):

Screen Shot 2022-01-19 at 3 30 48 PM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by AbhishekJamhoriya**
Signed-off-by: AbhishekJamhoriya <jamhoriya.1@iitj.ac.in>

**Description**
Add the word "BETA" as superscript to "MeshMap"
This PR fixes #2515

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Meshmap/meshmap.style.js [hunk 1]
```diff
@@ -2,15 +2,21 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    /* background-color: red; */
```
Comment (Chadha93) [inline src/sections/Meshmap/meshmap.style.js:6]: Remove this, don't comment out when not required directly remove it.

File: src/sections/Meshmap/meshmap.style.js [hunk 2]
```diff
@@ -2,15 +2,21 @@ import styled from "styled-components";
...
+  }
   .required-sign {
     color: #8b0000;
   }
 
   .checkbox-label {
     font-size: .8rem;
   }
-
+
```
Comment (Chadha93) [inline src/sections/Meshmap/meshmap.style.js:19]: Clear this space if added.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@AbhishekJamhoriya good update some minors to resolve + some merge conflicts to resolve :)

File: src/sections/Meshmap/meshmap.style.js (hunk 1 above)
Comment (AbhishekJamhoriya) [reply src/sections/Meshmap/meshmap.style.js:6]: ok, I will do that.

File: src/sections/Meshmap/meshmap.style.js (hunk 2 above)
Comment (AbhishekJamhoriya) [reply src/sections/Meshmap/meshmap.style.js:19]: got it.

**Review by AbhishekJamhoriya — COMMENTED** [review]
Signed-off-by: AbhishekJamhoriya jamhoriya.1@iitj.ac.in

**Comment by AbhishekJamhoriya** [comment]
Hi, @Chadha93  made some changes. please check it now.

**Review by Chadha93 — COMMENTED** [review]
@AbhishekJamhoriya Thanks for the update, merge conflict and dco to look into...

**Comment by warunicorn19** [comment]
I'll take care of the DCO, could you please solve the merge conflict.

**Comment by Chadha93** [comment]
> I'll take care of the DCO, could you please solve the merge conflict.

@warunicorn19 yes we could take care of the DCOs but if contributors solve these errors their Git skills will increase :P

**Comment by AbhishekJamhoriya** [comment]
Totally agree with @Chadha93 . Let me resolve both the problems.

**Comment by warunicorn19** [comment]
The only reason I set the DCO to pass on this PR is because the signoff wont be added to this PR even with further commits, cause it wasn't configured when you first made the PR. You'd have to open a new PR with a signoff configured to your system. 

@Chadha93 i completely agree with you, but we can be a little lenient on this repo.

**Comment by warunicorn19** [comment]
@AbhishekJamhoriya please go ahead and create a new PR.

**Comment by l5io** [comment]
🚀 Preview for commit 03b3a8366c6aec27777b9ac55bf90bc59cae3bba at: https://61ec9ad949f0b3a9106fbec3--layer5ng.netlify.app

**Comment by Chadha93** [comment]
@AbhishekJamhoriya you could signoff your previous commits within the same pr by using `git rebase --signoff HEAD~n` (n) is the number of commits.

**Pull request closed without merging**